src/hierarchies/updaters/abstract_updater.h

```diff
@@ -8,8 +8,12 @@ class AbstractUpdater {
  public:
   virtual ~AbstractUpdater() = default;
   virtual bool is_conjugate() const { return false; };
-  virtual void draw(AbstractLikelihood &like, AbstractPriorModel &prior,
-                    bool update_params) = 0;
+  //   virtual void draw(AbstractLikelihood &like, AbstractPriorModel &prior,
+  //                     bool update_params) = 0;
+  virtual void compute_posterior_hypers(AbstractLikelihood &like,
+                                        AbstractPriorModel &prior) {
+    throw std::runtime_error("compute_posterior_hypers not implemented");
+  }
 };
 
 #endif  // BAYESMIX_HIERARCHIES_UPDATERS_ABSTRACT_UPDATER_H_
```

Comment:
In this review I'm not finding the implementation of the draw() method in this new context. Can you please point me out where it is implemented? Thank you!